.go-env.sh

```diff
@@ -4,12 +4,25 @@ set -Eeuo pipefail
 dir="$(dirname "$BASH_SOURCE")"
 dir="$(readlink -ve "$dir")"
 
-user="$(id -u):$(id -g)"
+src="$dir"
+dst="$dir"
+msys=
+if [ "$(uname -o)" = 'Msys' ];	then
+	msys=1
+	if command -v cygpath > /dev/null; then
+		src="$(cygpath.exe --windows "$dst")"
+	fi
+fi
+windowsContianers=
```

Comment:
Containers is misspelled, but you do it thrice the same way so maybe I am missing something.

Reply:
`%s///` makes sure I am at least consistently wrong. Sometimes my fingers get ahead of each other.